configs/webpack/base.coffee

```diff
@@ -153,17 +139,23 @@ makeProductionBase = (projectConfig) ->
 
   # rename to minified
   output.filename = '[name].min.js'
-  # output.path = "#{projectConfig.basePath}/dist"
   {styleFilename} = projectConfig
   styleFilename ?= '[name].min.css'
 
   productionBase =
     output: output
+    devtool: 'source-map'
     module:
-      loaders: BASE_BUILD_LOADERS
+      rules: BASE_BUILD_LOADERS
     plugins: makeBuildPlugins({styleFilename}).concat([
       # Minify
-      new webpack.optimize.UglifyJsPlugin(minimize: true)
+      # https://webpack.github.io/docs/list-of-plugins.html#uglifyjsplugin
+      # https://github.com/webpack/webpack/issues/2704
+      new webpack.optimize.UglifyJsPlugin(
+        mangle: false
+        comments: false
+        sourceMap: true
```

Comment:
yay! I didn't realize that needed to be specified as an arg to Uglify, for some reason I thought webpack's sourcemap flag would work